src/main/groovy/io/openliberty/tools/gradle/LibertyTasks.groovy

```diff
@@ -38,7 +38,9 @@ public class LibertyTasks {
         project.libertyRun {
             dependsOn 'libertyCreate'
 
-            if (dependsOnApps(project.liberty.server)) dependsOn 'deploy'
+            if (dependsOnApps(project.liberty.server)) {
```

Comment:
General comment to update the copyright year in all the groovy files you updated.